portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
```

Comment:
If you want the server to persist the messages, other than logging them, you can store them in a private variable in the FormHandlerServlet class. You can also use the datastore Google Cloud provides to store data(in week 3 library samples), and that will persist when you restart the server.